internal/syncer/connector/jira.go

```
@@ -0,0 +1,935 @@
+func (c *JiraConnector) Validate(ctx context.Context) error {
+	if c == nil {
+		return &ConnectorValidationError{Message: "Jira connector is nil"}
+	}
+	if c.baseURL == "" {
+		return &ConnectorValidationError{Message: "Jira base URL must be provided."}
+	}
+	parsed, err := url.Parse(c.baseURL)
+	if err != nil || parsed.Scheme == "" || parsed.Host == "" {
+		return &ConnectorValidationError{Message: "invalid Jira base URL"}
+	}
+	if c.apiToken == "" && !(c.userEmail != "" && c.password != "") {
```

golangci-lint (2.12.2)

[error] QF1001: could apply De Morgan's law

(staticcheck)